shopify/standalone-shopify-template/web/routes/change-password.tsx

```diff
@@ -46,20 +49,19 @@ export default function () {
               control={control}
               render={({ field: { ref, ...fieldProps } }) => (
                 <TextField
+                  label="New Password"
+                  autoComplete="new-password"
                   placeholder="New password"
                   type="password"
                   {...fieldProps}
+                  value={fieldProps.value ?? ""}
                   error={errors?.user?.newPassword?.message}
                 />
               )}
             />
-            {errors?.root?.message && (
-              <Text as="p" className="format-message error">
-                {errors.root.message}
-              </Text>
-            )}
+            {errors?.root?.message && <Text as="p">{errors.root.message}</Text>}
             <Link to="/signed-in">Back to profile</Link>
-            <Button disabled={isSubmitting} type="submit" onClick={submit}>
+            <Button disabled={isSubmitting} submit onClick={submit}>
```

Comment:
```suggestion
            <Button disabled={isSubmitting} submit>
```
I don't think we need this onClick if the Form has an onSubmit handler and the button type is submit?

Reply:
This should also be updated (and tested) for `web/routes/forgot-password` and `web/routes/reset-password`

Reply:
I changed it in a few places